lib/libsys/thr_exit_c18n.c

```diff
@@ -25,15 +25,17 @@
  * SUCH DAMAGE.
  */
 
+#ifdef CHERI_LIB_C18N
 #include <sys/cdefs.h>
 
-void _rtld_thr_exit(long *);
-
-__weak_reference(__sys_thr_exit, thr_exit);
-__weak_reference(__sys_thr_exit, _thr_exit);
```

Comment:
Does this need `__weak_reference(thr_exit, _thr_exit)`?

Reply:
Ack.